openrouteservice/src/test/java/org/heigit/ors/api/util/SystemMessageTest.java

```diff
@@ -0,0 +1,36 @@
+package org.heigit.ors.api.util;
+
+import org.heigit.ors.api.requests.common.APIEnums;
+import org.heigit.ors.api.requests.isochrones.IsochronesRequest;
+import org.heigit.ors.api.requests.matrix.MatrixRequest;
+import org.heigit.ors.api.requests.routing.RouteRequest;
+import org.heigit.ors.exceptions.ParameterValueException;
+import org.heigit.ors.routing.RoutingRequest;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class SystemMessageTest {
+    @Test
+    public void testGetSystemMessage() throws ParameterValueException {
+        System.setProperty("ors_app_config", "../openrouteservice-api-tests/conf/app.config.test");
```

Comment:
I rather think that that is the appropriate way, instead many of our older unit tests depend on values set in a local app.config file that does not exist and you have to set up ORS_APP_CONFIG to get them to run. It would be better to fix all unit tests so that they run on the basis of the same file everywhere. 
That config.test referenced is afaik also the one used to run the service for the api tests, which is why I think the tests run fine on travis. The other option would be to have a app.config for tests in the openrouteservice project, but that would mean a third file to keep up to date every time you change something.

Reply:
I believe there is no good solution at the moment. The whole setup with app.conf and testing is a mess. Cleaning this up would be a further large project. Using ORS_APP_CONFIG would make the tests depend on a volatile environment, therefore, I second @takb 's solution for now.

Reply:
Did you check about the issue with that test failing? As it stands I still cannot build openrouteservice without removing the test

Reply:
I just fixed the old unit tests so that they rely on the same app.config.test from api-tests. Now all tests run as a whole without setting ORS_APP_CONFIG.

Reply:
So, I added another small improvement to the unit tests by telling maven to copy the app.config.test from openrouteservice-api-tests. now the uni tests run from maven and directly through intelli without requiring ORS_APP_CONFIG. For API tests locally you can still use ORS_APP_CONFIG for your local setup. I even suspect we could remove the copying command form the Travis setup, but it runs as is.